<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>812</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>333</width>
     <height>200</height>
    </rect>
   </property>
   <property name="title" >
    <string/>
   </property>
   <layout class="QGridLayout" name="gridLayout" >
    <item row="0" column="0" >
     <layout class="QVBoxLayout" name="verticalLayout_3" >
      <item>
       <widget class="QCheckBox" name="UseSizes" >
        <property name="text" >
         <string>Use Original sizes</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" >
        <item>
         <widget class="QLabel" name="label_9" >
          <property name="text" >
           <string>Block Sizes (Arrays)</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8" >
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5" >
            <item>
             <widget class="QLabel" name="label_8" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>30</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>30</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text" >
               <string>XINC</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="XINC" >
              <item>
               <property name="text" >
                <string/>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6" >
            <item>
             <widget class="QLabel" name="label_7" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
                <horstretch>30</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>30</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>30</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text" >
               <string>YINC</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="YINC" >
              <item>
               <property name="text" >
                <string/>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7" >
            <item>
             <widget class="QLabel" name="label_4" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>30</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>30</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text" >
               <string>ZINC</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="ZINC" >
              <item>
               <property name="text" >
                <string/>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" >
        <item>
         <widget class="QLabel" name="label_UsesSizes" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="text" >
           <string>Use Sizes (Values)</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4" >
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" >
            <item>
             <widget class="QLabel" name="label_10" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>16</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>16</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text" >
               <string>X</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="SizeCX" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="value" >
               <double>5.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" >
            <item>
             <widget class="QLabel" name="label_11" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>16</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>16</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text" >
               <string>Y</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="SizeCY" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="value" >
               <double>5.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" >
            <item>
             <widget class="QLabel" name="label_5" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>16</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>16</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text" >
               <string>Z</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="SizeCZ" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="value" >
               <double>5.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1" >
     <spacer name="horizontalSpacer" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType" >
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>13</width>
        <height>150</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" >
     <spacer name="verticalSpacer" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType" >
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>288</width>
        <height>18</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
